modxcms/revolution
# Fix date TV parsing bug when time is hidden

### Why is it needed?
Dates without time were not rendering correctly for TVs

[Some more context here](https://github.com/modxcms/revolution/issues/16393#issuecomment-1442461134).

### How to test
1. Create a new date TV with "Hide Time Option" enabled, assign it to a Template, and enter/save values in a Resource to verify correct behavior.
2. Edit an existing date TV (with time enabled, and having already assigned/saved a value in your test Resource), enabling "Hide Time Option." Verify that the TV's saved date still shows as expected.

### Related issue(s)/PR(s)
Resolves #16393
